.github/workflows/sync-translated-content.yml

```diff
@@ -64,3 +64,4 @@ jobs:
           title: "[${{ matrix.lang }}] sync translated content"
           author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
```

Comment:
should we also update the author (as `mdn-bot`)?

Reply:
https://github.com/mdn/translated-content/pull/8378#discussion_r969735385

Reply:
This `PAT` should belong to mdn-bot (I think)

Reply:
I don't think it would hurt, but I don't think it needs to match. I think the PAT is just used for the API access

Reply:
PR will be created by the owner of the `PAT`. So when we use the `mdn-bot`'s PAT, the PRs' creator will not be `github-actions[bot]` but `mdn-bot`. And see the behavior in mdn/content [before](https://github.com/mdn/content/commit/35b42668da5a200496a7f2214f6729fab392367e), the git committer will be `github-actions[bot]`.

See the second note: <https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#workarounds-to-trigger-further-workflow-runs>

Reply:
```suggestion
          author: mdn-bot <108879845+mdn-bot@users.noreply.github.com>
```